source/integration_and_development/system_interfaces/java.rst

```diff
@@ -9,14 +9,45 @@ This page contains the OpenNebula Cloud API Specification for Java. It has been
 Download
 ================================================================================
 
-You can download the ``.jar`` file compiled using Java 1.8, the required libraries, and the javadoc packaged in a tar.gz file `following this link <http://downloads.opennebula.io/packages>`__ in the OpenNebula version you have installed.
+You can download the ``.jar`` file compiled using Java 11, the required libraries, and the javadoc packaged in a tar.gz file `following this link <http://downloads.opennebula.io/packages>`__ in the OpenNebula version you have installed.
 
 You can also consult the `javadoc online </doc/6.4/oca/java/>`__.
 
+Compilation on Java 8
```

Comment:
can we say this is for older versions, instead of referring to a specific one? maybe we can make a note afterwards that we have tested this steps in Java 8